app/controllers/users/registrations_controller.rb

```diff
@@ -10,9 +10,11 @@ class Users::RegistrationsController < Devise::RegistrationsController
   # end
 
   # POST /resource
-  # def create
-  #   super
-  # end
+  def create
+    super
+    settings = User::Settings.new(resource)
+    settings.setup_new_user!
```

Comment:
13. If every use has a `FacebookAccount` what is the advantage of having a separate DB table for it?
14. What is the advantage of having the settings stored as a JSONB, instead of regular boolean columns? If we had everything in `user` we could just use default column values and would not require a `setup_new_user!`